### ISSUE TITLE
[NativeTarget] Add method to read the build settings taking into account any xcconfig

### ISSUE THREAD

**Issue by fabiopelosin** [CocoaPods/Xcodeproj#180]
This should be adopted by CocoaPods for the inspection of the targets that will be integrated.

Related: https://github.com/CocoaPods/CocoaPods/issues/2409

**Comment by segiddins** [comment]
@samdmarshall I can't remember, have you or @alloy built a crazy config parser yet?

**Comment by samdmarshall** [comment]
Yes, but it was in python, so i don't think it would be of tremendous value here.
